Test_Plan/Test_Plan_1.txt

```diff
@@ -0,0 +1,127 @@
+Verify form for  “Login”-                Verify that form for  “Login” is there and works
+Verify tab “Rooms” –                     Verify that tab “Rooms” is there and works
+Verify tab “Report”-                     Verify that tab “Report” is there and works
+Verify tab “Branding”-                   Verify that tab “Branding” is there and works
+Verify tab “Notification”-               Verify that tab “Notification” is there and works
+Verify tab “Front Page”-                 Verify that tab “Front page” is there and works
+Verify tab “Logout”-                     Verify that tab “logout” is there and works
+
+
+1.1.2	Feature not to be tested
+These feature are not be tested because they are not included in the software requirement specs
+-	Hardware Interfaces
+-	Database logical
+-	Website Security and Performance
+
+
+1.2	Test Type
+In the projectwebshop B&B,  there’re 3 types of testing should be conducted.
+-	Manual Testing: Test manualy all app.
+-	API testing: Test all the APIs create for the software under tested
+-	Automation testing: Test some test cases whit Robot Framework
+
+
+1.3	Risk and Issues
+
+Risk	                                                Mitigation
+One tester on the whole project                Set Test Priority for each of the test activity.
+
+
+1.4	Test Logistics
+1.4.1	Who will test?
+The whole project will be testing by one tester.
+
+1.4.2	When will test occur?
+The tester will start the test execution when all the following inputs are ready
+-	Software is available for testing
+-	Test Specification is created
+-	Test Environment is built
+
+2	TEST OBJECTIVE
+The test objectives are to verify the Functionality of website B&B, the project should focus on testing the
+Booking process, testing both roles: admin and customer and verification of other important functions:
+login form for admin, all text fields, booking process for the customer, check what the role can see,
+do security testing  to guarantee all these operation can work normally in real business environment.
+
+3	TEST CRITERIA
+3.1	Suspension Criteria
+If the team members report that there are 40% of test cases failed,
+suspend testing until the development team fixes all the failed cases.
+
+3.2	Exit Criteria
+Specifies the criteria that denote a successful completion of a test phase
+-	Run rate is mandatory to be 100% unless a clear reason is given.
+-	Pass rate is 80%, achieving the pass rate is mandatory.
+
+4	RESOURCE PLANNING
+4.1	System Resource
+1.	Server	 - Web server which install Maven Server
+2.	Test tool	- Develop a Test tool which can auto generate the test result to the pre- defined form
+                  and automated test execution
+
+4.2	Human Resource
+No.	Member	                         Tasks
+1.	SQA member - tester	             Take in charge of quality assurance and testing
+
+
+5	SCHEDULE & ESTIMATION
+5.1	All project task and estimation
+
+Task	                          Members	                       Estimate effort
```

Comment:
Ovdje  mozda mozes dodati i test automation, te environment setup

Reply:
Dodala sam, nadam se da je to,to.

Reply:
super